### DISCUSSION TITLE
配置无法使用，谷歌打不开都

### DISCUSSION THREAD

**Discussion by rr828** [Semporia/TikTok-Unlock#350] [General]
作者您好，今日配置无法打开谷歌等搜索引擎Z…是不是配置出现问题了

**Comment by Ledrong** [comment]
你好，我是來自俄羅斯的學生。我想在https://buff.163.com/平台進行交易。但為此我需要驗證我的帳戶，而這只能使用支付寶和steam來完成。請您只需要驗證一下，一切都是安全的。我會付1000元
